### ISSUE TITLE
Discrepancy with Ruby Sass for CSS with 0\0 media query hack

### ISSUE THREAD

**Issue by cvrebert** [sass/libsass#1219]
The hack in question: http://browserhacks.com/#hack-a60b03e301a67f76a5a22221c739dc64

Example:
```css
/* one.css */
@media screen and (min-width:0\0) {
    .bar {
        width: 12px;
    }
}
```

```
$ sass -v
Sass 3.4.13 (Selective Steve)
$ sass one.css 
Error: Invalid CSS after "...min-width:0\0) ": expected media query list, was "{"
        on line 1 of one.css
  Use --trace for backtrace.
```

```
$ sassc -v
sassc: 3.2.1
libsass: 3.2.4
sass2scss: 1.0.3
$ sassc one.css 
@media screen and (min-width: 0 \0) {
  .bar {
    width: 12px; } }
```

**Comment by xzyfer** [comment]
Thanks for the report @cvrebert. There some case where Libsass is more forgiving that Ruby Sass. It's important for us to be aware of these cases.

**Comment by xzyfer** [comment]
IMO Ruby Sass shouldn't error here since this is relatively common, and works in CSS. In either case we're producing the wrong output.

**Comment by cvrebert** [comment]
Yeah, I agree it would be preferable for Ruby Sass to not error out. I'm just not a Sass guru, so I thought I'd file a discrepancy report here to get some investigation rolling.

**Comment by xzyfer** [comment]
We appreciate it. I think it'd be worth opening an issue to Ruby Sass to see if the error case can be addressed upstream.

**Comment by cvrebert** [comment]
Okay, filed https://github.com/sass/sass/issues/1726

**Comment by xzyfer** [comment]
Legend.

**Comment by cvrebert** [comment]
It appears that sass/sass#1726 was not read sufficiently carefully and was thus misunderstood & erroneously closed by the maintainer.

**Comment by cvrebert** [comment]
Okay, per https://github.com/sass/sass/issues/1726#issuecomment-109444372 this does seem to be a SassC bug rather than a Ruby Sass bug.

**Comment by mgreter** [comment]
OK, we have no spec for this and if I run the original issue with latest ruby sass I get some output! Therefore it seems we should re-add the spec test and close this issue!

**Comment by cvrebert** [comment]
> if I run the original issue with latest ruby sass I get some output!

@mgreter Hmm?

```
$ sass -v
Sass 3.4.16 (Selective Steve)
$ sass one.css 
Error: Invalid CSS after "...min-width:0\0) ": expected media query list, was "{"
        on line 2 of one.css
  Use --trace for backtrace.
$
```

**Comment by mgreter** [comment]
OK, I guess someone should point out that the error is about using the file-ending `.css` with scss content. Ruby sass seems to default to sass (indented) in that situation. Not sure if we can replicate that behavior easily, since we depend on sass2scss for this kind of syntax.

**Comment by mgreter** [comment]
Changing this is rather easy (at least on the "surface") and leads to a similar error:
```
Error: invalid property name
        on line 2 of foo.css
>> @media screen and (min-width:0\0) { {
   -----------------------------------^
```

The "strange" output (double bracket) is due to the processing by `sass2scss`!
